diff --git a/config/config.go b/config/config.go
index 87d183a..489d28f 100644
--- a/config/config.go
+++ b/config/config.go
@@ -453,9 +453,9 @@ func LoadConfigFromFile(root *string, sharedir string) (*AercConfig, error) {
 		root = &_root
 	}
 	filename := path.Join(*root, "accounts.conf")
-	if err := checkConfigPerms(filename); err != nil {
-		return nil, err
-	}
+	// if err := checkConfigPerms(filename); err != nil {
+	// 	return nil, err
+	// }
 	filename = path.Join(*root, "aerc.conf")
 	file, err := ini.LoadSources(ini.LoadOptions{
 		KeyValueDelimiters: "=",

